src/app.rs

```diff
@@ -2165,6 +2165,51 @@ impl Application for App {
                                 }
                             }
                         }
+                        tab::Command::OpenMultipleFiles(items) => {
```

Comment:
There should be a fallback to the original open code if no default apps are found, which will always happen on OSes like macOS and Windows.

Reply:
Oops, did not think about it. Of course I will make this change :)